tests/tests_pytorch/strategies/test_bagua_strategy.py

```diff
@@ -45,6 +45,7 @@ def test_bagua_default(tmpdir):
     assert isinstance(trainer.strategy, BaguaStrategy)
 
 
+@pytest.mark.skip("Internal error in Bagua")  # Bagua stopped working: 'AssertionError: Unexpected rsp=<Response [500]'
```

Comment:
I suggest using xfail instead so that we know if it gets fixed 

```suggestion
@pytest.mark.xfail(raises=AssertionError, reason="Internal error in Bagua")  # Unexpected rsp=<Response [500]'
```